<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>802</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>42</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>611</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">settabtext:Horizontal;</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabWidgetPage1" native="true">
     <attribute name="title">
      <string>标准型</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>311</width>
        <height>481</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_ac">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>AC</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_00">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>+/-</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_per">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>110</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>％</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_dvd">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>390</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>45</pointsize>
        </font>
       </property>
       <property name="text">
        <string>÷</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>42</pointsize>
        </font>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>180</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>42</pointsize>
        </font>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_mul">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>320</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>45</pointsize>
        </font>
       </property>
       <property name="text">
        <string>×</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>250</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>42</pointsize>
        </font>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>42</pointsize>
        </font>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>180</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>42</pointsize>
        </font>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>250</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>42</pointsize>
        </font>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_min">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>250</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>42</pointsize>
        </font>
       </property>
       <property name="text">
        <string>﹣</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_add">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>180</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>42</pointsize>
        </font>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_0">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>390</y>
         <width>81</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>42</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>320</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>42</pointsize>
        </font>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>320</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>42</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>320</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>42</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_dot">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>390</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>42</pointsize>
        </font>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_eq">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>390</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>42</pointsize>
        </font>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>291</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_back">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>110</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/icon/backword.png</normaloff>:/icon/backword.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>291</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="lineEdit_hint">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>510</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabWidgetPage2" native="true">
     <attribute name="title">
      <string>科学型</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>pushButton_7</tabstop>
  <tabstop>pushButton_8</tabstop>
  <tabstop>pushButton_9</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButton_1</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_0</tabstop>
  <tabstop>pushButton_dot</tabstop>
  <tabstop>pushButton_eq</tabstop>
  <tabstop>pushButton_ac</tabstop>
  <tabstop>pushButton_00</tabstop>
  <tabstop>pushButton_per</tabstop>
  <tabstop>pushButton_back</tabstop>
  <tabstop>pushButton_add</tabstop>
  <tabstop>pushButton_min</tabstop>
  <tabstop>pushButton_mul</tabstop>
  <tabstop>pushButton_dvd</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_hint</tabstop>
 </tabstops>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
